src/app/app.module.ts

```diff
@@ -70,7 +72,11 @@ export type StoreType = {
     HttpModule,
     PublicModule,
     SpaceDialogModule,
-    AppRoutingModule
+    AppRoutingModule,
```

Comment:
The AppRoutingModule needs to be the last thing in the list, everything else is usually listed alphabetically.